src/components/Cards/Cards.component.jsx

```diff
@@ -0,0 +1,24 @@
+import React, { Fragment } from 'react';
+import Card from '../../components/Card';
+
+function Cards(props) {
+  return (
+    <Fragment>
```

Comment:
**Optional**:

we can use `<>` instead of `<Fragment>` as a less verbose of accomplishing the same thing